src/components/LaunchDetailsPage/selectors/selectors.ts

```diff
@@ -0,0 +1,11 @@
+import { createSelector } from '@reduxjs/toolkit';
```

Comment:
Do we use createSelector from redux-toolkit?

Reply:
@topsart, is it ok? I remember you approved using of  createSelector from Reselect library which reexports by @reduxjs/toolkit